src/vcSession.cpp

```diff
@@ -102,7 +102,7 @@ void vcSession_Login(void *pProgramStatePtr)
   vdkError result;
   vcState *pProgramState = (vcState*)pProgramStatePtr;
 
-  result = vdkContext_Connect(&pProgramState->pVDKContext, pProgramState->settings.loginInfo.serverURL, udTempStr("Euclideon Vault Client / " TAG_SUFFIX " (%s)", vcSession_GetOSName()), pProgramState->settings.loginInfo.username, pProgramState->password);
+  result = vdkContext_Connect(&pProgramState->pVDKContext, pProgramState->settings.loginInfo.serverURL, "Euclideon Vault Client / " TAG_SUFFIX, pProgramState->settings.loginInfo.username, pProgramState->password);
```

Comment:
Was `vcSession_GetOSName()` used for anything else? Could probably just remove it?

Reply:
Get Packages; I did remove it but it turns out it was being used...